src/implementations/twig/components/site-header-standardised/site-header-standardised.story.js

```diff
@@ -86,6 +91,20 @@ const getArgTypes = () => {
       category: 'Content',
     },
   };
+  argTypes.cta_link = {
+    name: 'call to action',
+    type: { name: 'boolean' },
+    defaultValue: false,
```

Comment:
Same as before, the object we build in `getArgs()` is replacing the `defaultValue` for each control as we were used to set them.